01-intro-til-weben/04-skyen.md

```diff
@@ -0,0 +1,76 @@
+
+Hvilken plattform man ønsker å lagre det på kan man velge selv, men i Bekk bruker vi ofte [BitBucket](https://bitbucket.org/), [GitHub](https://github.com/) eller [VSTS](https://visualstudio.microsoft.com/team-services/). I denne oppgaven lenker vi til ressurser for å få det på GitHub, men du står fritt til å velge hvilken remote du ønsker.
+
+#### 1. Opprett et nytt repository
+
+Når du er innlogget og står på forsiden til [GitHub](https://github.com/), trykk på den grønne knappen det står "New repository" på. **Obs!** pass på å gjøre det på din private bruker, og med mindre du betaler for den, kommer repoet til å være public og tilgjengelig for hvem som helst. Kall repositoryet det du vil. Siden vi tenker å legge deres første nettside der, trenger dere heller ikke å initialisere den med `README` eller `.gitignore`.
+
+Vi har lyst til å dytte nettsiden du begynte på i [første kapittel](01-intro-til-weben/01-hello.md) til git. Siden vi allerede har mappen, vil vi gjøre den om til et git-repository som beskrevet under _…or create a new repository on the command line_:
+
+#### 2. Først legger vi til en README med bare en header i repoet:
+
+```sh
+echo "# portfolio" >> README.md
+```
+
+#### 3. Også greit å ha med en `.gitignore`
+
+```sh
+echo ".DS_Store" >> .gitignore
+```
+
+I `.gitignore` legger vi filer vi ikke trenger å ta vare på, typisk midlertidige filer som `.DS_Store` som dukker opp på macer i hytt og pine, `node_modules` som genereres når man kjører kommandoer som `npm install` osv. Vær alltid obs på om det har dukket opp filer du ikke trenger å versjonere når du commiter til git! Følgende verdier kan du legge inn i `.gitignore` nå med en gang:
+
+```
+.DS_Store
+.idea/
+\_node_modules/
+*.log
+```
+
+#### 4. Så initialiserer vi mappen med git og legger til alle filene:
+
+```sh
+git init
+git add .
+```
+
+#### 5. Vi må lage en initiell commit
+
+```
+git commit -m "first commit"
+git remote add origin
+git@github.com:[BRUKERNAVN]/[REPONAVN].git
+git push -u origin master
+```
+
+Urlen til repositoryet ditt står på siden du er på allerede, mest sannsynlig. Når du har pushet koden opp, har du offisielt introdusert versjonering på prosjektet ditt!
+
+## Publiser koden på Heroku
+
+Lag deg en bruker på [Heroku](heroku.com). Heroku er en skyplattform som er ganske enkel å komme igang med, og som vi skal lansere applikasjonen vår på.
+
+> Det finnes mange andre gode skyleverandører der ute, ikke minst de tre store: AWS, Google Cloud og Azure. Når det gjelder små hobbyprosjekter, er Heroku og [Firebase](https://firebase.google.com) to veldig enkle og gode alternativer.
+
+#### 1. Lag en ny app
+
+I høyre hjørne når du har logget deg inn, er det en knapp med teksten "New". Trykk på denne og velg "Create new app". Velg et navn som ikke finnes allerede - det er ikke veldig farlig akkurat hva den heter. Du kan enkelt sette opp siden med et [domene du har kjøpt](domeneshop.com) senere.
```

Comment:
*logget deg inn på Heroku

Tenker det kan være greit å spesifisere siden mye over har gått på git :)

Reply:
Forslag til endring siste setning:

"Du kan enkelt sette opp siden med et domene du har kjøpt. Det finnes mange tilbydere for domener og hosting, som domeneshop.com, one.com, GoDaddy og mange, mange fler"